tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -26,12 +26,26 @@ jobs:
         msbuildArchitecture: 'x64'
         logProjectEvents: true
         workingFolder: '$(Build.BinariesDirectory)\Debug'
+    
     - task: BatchScript@1
       displayName: 'Test Debug'
       inputs:
         filename: '$(Build.BinariesDirectory)\packages\python\python.exe'
         arguments: '$(Build.SourcesDirectory)\tools\ci_build\build.py --config Debug --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe --use_tvm --enable_pybind --use_mkldnn --use_mklml --use_openmp --build_shared_lib  --enable_onnx_tests --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum) --gen_doc --test'
         workingFolder: '$(Build.BinariesDirectory)'
+    
+    - task: PowerShell@2
+      displayName: 'Run Test Coverage'
+      inputs:
+        filePath: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\run_OpenCppCoverage.ps1'
+        arguments: '-OpenCppCoverageExe:"$(Build.BinariesDirectory)\OpenCppCoverage\OpenCppCoverage.exe" -SourceRoot:"$(Build.SourcesDirectory)" -BuildRoot:"$(Build.BinariesDirectory)" -LocalBuild:$false'
+
+    - task: PublishCodeCoverageResults@1
+      inputs:
+        codeCoverageTool: 'Cobertura'
+        summaryFileLocation: '$(Build.BinariesDirectory)/**/cobertura.xml'
+        reportDirectory: '$(Build.BinariesDirectory)/**/OpenCppCoverageResults'
+    
```

Comment:
This section this can be modularized into a step-template, so  it doesn't don't have to repeated in multiple pipelines (e.g. x86, mkml and other pipelines in future). Can this be in  template/win-code-coverage.yml instead?

Reply:
In fact we don't plan to run this for multiple pipeline as of now. Having a test coverage for the win-CPU pipeline would give us a good indicator of overall coverage.

Reply:
The no-contrib-ops pipeline will have a lot of tests disabled (so maybe code-coverage will be quite different there). Another reason is to keep the win-ci-template file as compact as possible. For example, if we add SDL compliance tasks, as well as code-coverage tasks here, it will bloat further.